src/frame/initialization/tests/index.test.ts

```diff
@@ -4,13 +4,18 @@ import * as TestUtils from "../../../tests/TestUtils";
 import * as FrameUtils from "../../FrameUtils";
 import Future from "futurejs";
 import * as InitializationApi from "../InitializationApi";
+import {publicKeyToBase64} from "../../../lib/Utils";
+import {fromByteArray} from "base64-js";
 
 describe("init index", () => {
-    describe("initialize", () => {
-        afterEach(() => {
-            localStorage.clear();
-        });
+    beforeEach(() => {
+        (ApiState as any).apiUser = undefined;
```

Comment:
Should we maybe use `ApiState.clearCurrentUser()` instead of accessing the private property or did you need to only clear that one field?

Reply:
Yes, and I considered that at the time. I wasn't sure why the code was doing what it was doing in the first place and not that. I was just abstracting this up to a `beforeEach` because most of the tests either were doing this or should have been doing this.